install/setup-js-sdk-assets.sh

```diff
@@ -0,0 +1,37 @@
+# This will only run if the SETUP_JS_SDK_ASSETS environment variable is set to 1
+if [[ "${SETUP_JS_SDK_ASSETS:-}" == "1" ]]; then
```

Comment:
IMO if we're going to include this as part of the install script moving forwards, I'd like to see it in the `.env` file just to surface this as something people can set up. We can leave it as default for opt-in

Reply:
Oh not you too :(

Reply:
Oh sorry I read that wrong. Lol.

Reply:
Okay done